### ISSUE TITLE
AI doesn't draw weapons

### ISSUE THREAD

**Issue by Lexx2k** [alexbatalov/fallout2-re#23]
[SLOT04.zip](https://github.com/alexbatalov/fallout2-re/files/8647675/SLOT04.zip)

Seen it in a couple situations already. They have weapons in their inventory, but they don't use them. Sometimes the ai will not move at all due to this, other times they just attack unarmed.

/Edit: Sulik also doesn't pull out his weapon anymore.

**Comment by alexbatalov** [comment]
Order Sulik to use best weapon. This would fix his fid.

**Comment by Lexx2k** [comment]
I just tested latest build, enemy ai still not drawing any weapons. I cleared out a cave full of raiders with my little spear:
[SLOT05.zip](https://github.com/alexbatalov/fallout2-re/files/8649233/SLOT05.zip)
 
Easy to reproduce, just walk around the raiders location until you get a raiders encounter. None of them will pull out a weapon.

/Edit: Of course now that I wrote this, I see encounters with critters and their weapons out ...
